### PR TITLE
Added some starkit related tests

### PR DESCRIPTION
**Pull request by 97amarnathk**
Added basic tests for 2 starkit utility functions:
* `fwhm2sigma`
* `sigma2fwhm`

### CODE REVIEW

File: starkit/example_c.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+def primes(n):
```
Comment (dchu808) [inline starkit/example_c.py]: These are example files, please remove them

File: starkit/example_mod.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+def primes(n):
```
Comment (dchu808) [inline starkit/example_mod.py]: Same applies here

File: starkit/tests/test_spectral.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+from starkit.utils import spectral
+
+def test_fwhm2sigma():
+    assert spectral.fwhm2sigma(spectral.FWHM2SIGMA_CONST) == 1.0
```
Comment (dchu808) [inline starkit/tests/test_spectral.py]: This is a smart way of doing it!

File: starkit/example_c.py (hunk 1 above)
Comment (97amarnathk) [reply starkit/example_c.py]: Yes. But if these are removed, the example tests will fail. So shall I remove the example tests as well?

File: starkit/tests/test_spectral.py (hunk 3 above)
Comment (97amarnathk) [reply starkit/tests/test_spectral.py]: Do you suggest any changes here? Or can it be merged?

File: .cache/v/cache/lastfailed [hunk 4]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (wkerzendorf) [inline .cache/v/cache/lastfailed]: can you purge these files completely from the history - they should not be added to the repository

File: .vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (wkerzendorf) [inline .vscode/settings.json]: same goes for this

**Comment by wkerzendorf** [comment]
@97amarnathk try to add `.travis.yml` to get the continuous integration going.

File: .travis.yml [hunk 6]
```diff
@@ -62,28 +23,19 @@ before_install:
     - wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
     - chmod +x miniconda.sh
     - ./miniconda.sh -b
-    - export PATH=/home/travis/miniconda/bin:$PATH
+    - export PATH=/home/travis/miniconda2/bin:$PATH
```
Comment (wkerzendorf) [inline .travis.yml]: why do you name it miniconda2?

File: .travis.yml [hunk 7]
```diff
@@ -62,28 +23,19 @@ before_install:
...
-    - source activate test
-
-    # CORE DEPENDENCIES
-    - if [[ $SETUP_CMD != egg_info ]]; then $CONDA_INSTALL numpy=$NUMPY_VERSION pytest pip Cython jinja2; fi
-    - if [[ $SETUP_CMD != egg_info ]]; then $PIP_INSTALL pytest-xdist; fi
-
-    # ASTROPY
-    - if [[ $SETUP_CMD != egg_info ]] && [[ $ASTROPY_VERSION == development ]]; then $PIP_INSTALL git+http://github.com/astropy/astropy.git#egg=astropy; fi
-    - if [[ $SETUP_CMD != egg_info ]] && [[ $ASTROPY_VERSION == stable ]]; then $CONDA_INSTALL numpy=$NUMPY_VERSION astropy; fi
+    - curl -O https://raw.githubusercontent.com/starkit/starkit/master/starkit_env27.yml
```
Comment (wkerzendorf) [inline .travis.yml]: you are in a cloned version of the repo - so you can just use the local starkit_env

File: .travis.yml [hunk 8]
```diff
@@ -62,28 +23,19 @@ before_install:
...
+    - conda env create -n starkit --file starkit_env27.yml
+    - source activate starkit
+    - pip install setuptools==36.2.0
```
Comment (wkerzendorf) [inline .travis.yml]: as this is a requirement - that should be added to the env file.

Comment (97amarnathk) [reply .travis.yml]: Okay.

File: .travis.yml (hunk 7 above)
Comment (97amarnathk) [reply .travis.yml]: Okay.

File: .travis.yml (hunk 6 above)
Comment (97amarnathk) [reply .travis.yml]: Current distributions of miniconda install by default to `miniconda2` instead of `miniconda`

Comment (wkerzendorf) [reply .travis.yml]: okay

**Comment by wkerzendorf** [comment]
well done - congratulations! your first starkit merge 👍

**Pull request merged**